Dockerfile.import

```diff
@@ -1,8 +1,24 @@
 FROM ubuntu:xenial
 
+ARG BUILD_OSM2PGSQL
+ARG OSM2PGSQL_VERSION
+
 RUN apt-get update && apt-get install --no-install-recommends -y \
-    curl ca-certificates osm2pgsql postgresql-client \
-    && rm -rf /var/lib/apt/lists/*
+    curl ca-certificates postgresql-client
```

Comment:
Not removing lists drastically inflates the image size here

Reply:
Re-enabled removing lists here.